controllers/replication-controller/dellcsireplicationgroup_controller_test.go

```diff
@@ -424,6 +426,21 @@ func (suite *RGControllerTestSuite) TestRGSyncDeletion() {
 	suite.Equal(false, resp.Requeue)
 }
 
+func (suite *RGControllerTestSuite) TestNoDeletionTimeStamp() {
```

Comment:
Is this testing a successful reconcile? In that case, can you rename the function name. For ex: TestRGSync?
It simply checks for no error at the end. Could you check for some annotations that was set on RG?